pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -100,8 +102,10 @@ func (c *VPCConfig) ExplainConnectivity(src, dst string, connQuery *common.Conne
 	}
 
 	return &Explanation{c, connQuery, &rulesAndDetails, src, dst,
-		getNetworkInterfacesFromIP(isSrcInternalIP, srcNodes),
-		getNetworkInterfacesFromIP(isDstInternalIP, dstNodes),
+		getNetworkInterfacesFromIP(
+			isSrcDstInternalIP == srcInternalIP || isSrcDstInternalIP == srcAndDstInternalIP, srcNodes),
+		getNetworkInterfacesFromIP(
+			isSrcDstInternalIP == dstInternalIP || isSrcDstInternalIP == srcAndDstInternalIP, dstNodes),
```

Comment:
can simplify this?

Reply:
indeed. typo (err1 instead of err) fixed